### ISSUE TITLE
unpack_from doesn't work on bytearray in python <=  2.7.3

### ISSUE THREAD

**Issue by alatdneg** [markreidvfx/pyaaf2#30]
Release 1.0.0 works, but not current master branch.
ta
```
Traceback (most recent call last):
  File "./test_pyaaf2.py", line 8, in <module>
    with aaf2.open("./b0000.aaf", "r") as f:
  File "/tools/SITE/python/aaf2/file.py", line 188, in __init__
    self.cfb = CompoundFileBinary(self.f, self.mode, sector_size=sector_size)
  File "/tools/SITE/python/aaf2/cfb.py", line 750, in __init__
    self.mini_stream_chain = self.get_fat_chain(self.root.sector_id)
  File "/tools/SITE/python/aaf2/cfb.py", line 434, in sector_id
    sid = struct.unpack_from(str('<I'), self.data, 116)[0]
TypeError: unpack_from() argument 1 must be string or read-only buffer, not bytearray

```

**Comment by alatdneg** [comment]
I've no idea why this isn't working as the code seems to imply it's a "str"

**Comment by alatdneg** [comment]
python --version
Python 2.7.3

**Comment by alatdneg** [comment]
Ah.. Bug in Python2.7.3 oh joy...

https://bugs.python.org/issue10212fixed in 2.7.4

**Comment by markreidvfx** [comment]
I gonna fixed this, even though its a bug in python 2.7.3.  Thanks for reporting, sorry for the breakage.

**Comment by markreidvfx** [comment]
This should be fixed in the lastest master now
